src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -1426,6 +1426,50 @@ public static bool EqualsAny<T>(Vector128<T> left, Vector128<T> right)
                 || Vector64.EqualsAny(left._upper, right._upper);
         }
 
+        internal static Vector128<T> Exp<T>(Vector128<T> vector)
```

Comment:
Is this used somewhere outside of this type, or could it be private?

Reply:
Copy/paste error. V128/256/512 should be calling the next smaller size on the upper/lower halves, as that gives the simplest implementation and tends to better performance on hardware that can't directly accelerate that size.

Log/Log2 are doing the right thing, just messed up here for `Exp`